libparsec/crates/platform_device_loader/tests/keyring.rs

```diff
@@ -0,0 +1,8 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+#![cfg(not(target_arch = "wasm32"))]
+
+#[test]
+fn keyring_available() {
+    assert!(libparsec_platform_device_loader::is_keyring_available())
```

Comment:
I don't like that we make the assumption that the keyring is installed on the dev env.

I think we don't need to test the method since it's a one liner to `keyring::is_available`

Reply:
Removed

Reply:
The comment should be enough to guarantee that someone won't change the value to empty string